core/src/main/java/org/mule/runtime/core/api/context/notification/PolicyNotification.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.api.context.notification;
```

Comment:
This should be in the pacakge org.mule.runtime.api.notification.policy in the mule-api proyect